.release/linux/package/etc/consul.d/consul.hcl

```diff
@@ -0,0 +1,96 @@
+# Fullconfiguration options can be found at https://www.consul.io/docs/agent/options.html
```

Comment:
I think it would be good to remove all the comments here and make it a minimal file if this is used exclusively for releasing we don't need the comments. Can be a follow up though.

Reply:
created a jira will follow up with Kyle and crt team